src/Microsoft.FeatureManagement/FeatureDefinition.cs

```diff
@@ -42,5 +42,27 @@ public class FeatureDefinition
         /// A list of variant definitions that specify a configuration to return when assigned.
         /// </summary>
         public IEnumerable<VariantDefinition> Variants { get; set; } = Enumerable.Empty<VariantDefinition>();
+
+        /// <summary>
+        /// A value used to group feature flags.
+        /// A <see cref="Label"/> is used together with a <see cref="Name"/> to uniquely identify a feature.
```

Comment:
The label is metadata to a feature flag. Our API doesn't support using a label (and name) to uniquely identify a feature, for example, `IsEnabledAsync("FlagName", "FlagLabel")`.